tests/README.md

```diff
@@ -103,6 +103,10 @@ Each runtime testcase consists of multiple directives. In no particular order:
 * `MIN_KERNEL`: Skip the test unless the host's kernel version is >= the
   provided kernel version. Try not to use this directive as kernel versions may
   be misleading (backported kernel features, for example)
+* `MAX_KERNEL`: Skip the test unless the host's kernel version is <= the
```

Comment:
Nit: shouldn't this addition be in the commit message as well?

Edit: Ah, you're just adding details about it. It already existed. Carry on.